libs/blocks/tabs/tabs.css

```diff
@@ -331,10 +415,50 @@
   background: transparent;
 }
 
+.tabs.radio.center div[role="tablist"] {
+  justify-content: center;
+}
+
+.tabs.radio.right div[role="tablist"] {
```

Comment:
I'm on fire with the RTL stuff today 😅 On RTL, _right_ would be left, are we using the same naming conventions in other blocks?

Reply:
@overmyheadandbody I agree with you on this, but it's unfortunately a convention that has been baked in everywhere, even in the Text block:

https://github.com/adobecom/milo/blob/bbdd381f9c3d178b80b4987b3ea50ec9862c43ad/libs/blocks/text/text.css#L95

Reply:
Oh, I see, thanks for the clarification! Then yes, it makes sense to keep it as-is for now for consistency.